app/views/speech/show.html.erb

```diff
@@ -0,0 +1,58 @@
+
+<% content_for :extra_headers do %>
+   <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', {
+        title: content_item.title,
+        context: I18n.t("csv_preview.document_type.#{content_item.document_type}", count: 1),
+        context_locale: t_locale_fallback("csv_preview.document_type.#{content_item.document_type}", count: 1),
+        average_title_length: "long",
+      } %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+  </div>
+</div>
+
+<%= render 'shared/publisher_metadata_with_logo' %>
```

Comment:
This has been renamed to `publisher_metadata` in https://github.com/alphagov/frontend/pull/4372/commits/c4dceb3647efdbd076884ab221886c7f51bd74d0